api/src/mutations/collection.rs

```diff
@@ -653,3 +717,14 @@ pub struct PatchCollectionPayload {
     /// The drop that has been patched.
     collection: CollectionObject,
 }
+
+#[derive(Debug, Clone, Serialize, Deserialize, InputObject)]
+pub struct ImportCollectionInput {
+    project_id: Uuid,
+    mint: String,
```

Comment:
```suggestion
    collection: String,
```

Add comment to the filed stating it's the mint address of the metaplex certified collection.